tests/pom.xml

```diff
@@ -74,6 +74,12 @@
     </dependency>
     <dependency>
       <groupId>org.tachyonproject</groupId>
+      <artifactId>tachyon-examples</artifactId>
```

Comment:
it doesn't seem right that tests should depend on examples. What is this needed for?